website/docs/community/community-docs/oss-expectations.md

```diff
@@ -10,7 +10,7 @@ Have you seen things in other OSS projects that you quite like, and think we cou
 
 ### Open source is participatory
```

Comment:
I wonder if this file should show up higher in the left-side TOC? So they know the expectations prior to reading about where they should contribute? Or a link to it?